docs/build.md

```diff
@@ -61,10 +61,10 @@ Epoch source code can be obtained by cloning the public [GitHub repository](http
 git clone https://github.com/aeternity/epoch.git epoch && cd epoch
 ```
 
-Then checkout the version to be build, e.g. `v0.12.0`:
+Then checkout the version to be build, e.g. `v0.13.0`:
```

Comment:
Updating the docs with version is annoying, maybe we should use at least bash var, or better wildcards :)